prac_07/project.py

```diff
@@ -0,0 +1,212 @@
+"""
+CP1404 - Lucas O'Brien
+Prac_07 - Project Management Program
+estimate - 90 minutes
+actual - 8 hours
```

Comment:
Just ran you're code and seems flawless and I put the error checking through its paces and didn't falter. Sorry took a while because Im at work atm.

Reply:
It was so unnecessarily extensive

Reply:
I nearly cried :')